File: tests/nodejs/vote/voteComment.js
var request = require('request');

var jsonBody = {
	vote: 1,
	comment_id: 1
}

request({
	url: 'http://localhost:3000/voteComment',
	method: 'post',
	json: true,
	body: jsonBody
}, function (error, response, body) {
	console.log(response);
});

File: tests/python/vote/voteComment.py
import requests
import json
import sys

jsonBody = {
	"comment_id": 1,
	"username": 'admin',
	"rating": 1
}

headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
res = requests.post('http://localhost:3000/voteComment', data = json.dumps(jsonBody), headers = headers)
print(res.json())